lib/twilio-ruby/rest/client.rb

```diff
@@ -67,11 +62,6 @@ def frontline_api
                 @frontline_api ||= FrontlineApi.new self
             end
             ##
-            # Access the Insights Twilio Domain
```

Comment:
Insight changes exists in library, I think we don't need to remove this.

Reply:
Re-instating it🙂